docs/web-apps/automated-testing/cypress.md

```diff
@@ -6,184 +6,148 @@ sidebar_label: Cypress
+                    *### .#######/  ##  / ########                  
+                   ###   .###########//###########                  
+               ######     ########################                  
+             (#(    *#(     #######.    (#######                    
+                    ##,    /########    ########                    
+                           *########    ########                    
+
+   _____        _    _  _____ ______    _____ _      ____  _    _ _____  
+  / ____|  /\  | |  | |/ ____|  ____|  / ____| |    / __ \| |  | |  __ \
+ | (___   /  \ | |  | | |    | |__    | |    | |   | |  | | |  | | |  | |
+  \___ \ / /\ \| |  | | |    |  __|   | |    | |   | |  | | |  | | |  | |
+  ____) / ____ \ |__| | |____| |____  | |____| |___| |__| | |__| | |__| |
+ |_____/_/    \_\____/ \_____|______|  \_____|______\____/ \____/|_____/
+14:30:22 INF Project archived. durationMs=3 size=5846
+14:30:23 INF Project uploaded. durationMs=1010 storageId=4ecb5962-f291-4e3e-9046-87c4783393c4767609/app.zip
+14:30:23 INF Launching workers. concurrency=2
+14:30:23 INF Starting suite. region=us-west-1 suite="Firefox in sauce"
+14:30:23 INF Starting suite. region=us-west-1 suite="Chrome using global mode setting"
+14:30:25 INF Suite started. browser=chrome platform="Windows 10" suite="Chrome using global mode setting" url=https://app.saucelabs.com/tests/d9c150834af0497db4e5097763dbf0bd
+14:30:27 INF Suite started. browser=firefox platform="Windows 10" suite="Firefox in sauce" url=https://app.saucelabs.com/tests/66f4be6117ef4d65b6314bb14050d59a
+14:30:33 INF Suites in progress: 2
+14:30:43 INF Suites in progress: 2
+14:30:53 INF Suites in progress: 2
+14:31:03 INF Suites in progress: 2
+14:31:12 INF Suite finished. passed=true suite="Firefox in sauce" url=https://app.saucelabs.com/tests/66f4be6117ef4d65b6314bb14050d59a
+14:31:13 INF Suites in progress: 1
+14:31:23 INF Suites in progress: 1
+14:31:33 INF Suites in progress: 1
+14:31:43 INF Suites in progress: 1
+14:31:53 INF Suites in progress: 1
+14:32:03 INF Suites in progress: 1
+14:32:13 INF Suites in progress: 1
+14:32:23 INF Suites in progress: 1
+14:32:26 INF Suite finished. passed=true suite="Chrome using global mode setting" url=https://app.saucelabs.com/tests/d9c150834af0497db4e5097763dbf0bd
+14:32:26 INF ┌───────────────────────┐
+14:32:26 INF  All suites have passed!
+14:32:26 INF └───────────────────────
+```
+
+The results and test assets are immediately available in the [Sauce Labs dashboard](https://app.saucelabs.com/dashboard/tests/vdc):
 
-6. Watch test runs in real-time, or verify the results in the [Sauce Labs dashboard](https://app.saucelabs.com/dashboard/tests/vdc):
-   
    <img src={useBaseUrl('img/cypress/test-results.png')} alt="Cypress Test Results" />
 
-:::warning Running Tests with a Local App
-
-If you plan to run your cypress tests with either of the following scenarios:
+## Advanced Considerations
```

Comment:
Would remove this section and everything below, just have a list of the links to the Cypress TRT docs